pkg/unittest/defailt_machine_deployment.go

```diff
@@ -0,0 +1,84 @@
+package unittest
```

Comment:
I can't change the typo directly but the filename should be `default_` not `defailt_`.

Reply:
good catch, will change it 👍